Sources/XcbeautifyLib/Pattern.swift

```diff
@@ -239,11 +239,11 @@ enum Pattern: String {
     /// Regular expression captured groups:
     /// $1 = file
     /// $2 = build target
-    case processPch = #"ProcessPCH(?:\+\+)?\s.*\s\/.*\/(.*.pch) normal .* .* .* \((in target: (.*)|in target '(.*)' from project '.*')\)"#
+    case processPch = #"ProcessPCH(?:\+\+)?\s.*\s\/.*\/(.*) normal .* .* .* \((in target: (.*)|in target '(.*)' from project '.*')\)"#
 
     /// Regular expression captured groups:
     /// $1 file path
-    case processPchCommand = #"\s*.*\/usr\/bin\/clang\s.*\s\-c\s(.*.pch)\s.*\-o\s.*"#
+    case processPchCommand = #"\s*.*\/usr\/bin\/clang\s.*\s\-c\s(.*?)(?<!\\)\s.*\-o\s.*"#
```

Comment:
Good point, thanks for bringing it up!

For `processPchCommand`, we can ensure the output file specified by `-o` ends in `.gch`, which is the file extension Xcode uses for precompiled header object files.

I'll update the PR

Reply:
One note: The updated regex would match *if* for some reason the .gch is referenced *after* the actual output object file.
Other compile commands using the PCH do not list the the .gch extension in the output, and that `include` also always comes before the `-o`, so that should never happen. Thus I chose to do it this way as the alternative regex would be more complicated and thus harder to read. If the alternative is preferred I can swap to that.